### DISCUSSION TITLE
is it possible to hide the cmd output?

### DISCUSSION THREAD

**Discussion by gmr458** [goolord/alpha-nvim#145] [Q&A] [answered]
this is when i run neovim, everything is fine
![mié 07 sep 2022 12:16:11 -05](https://user-images.githubusercontent.com/53343401/188940112-9af4a8c7-6074-4b14-bd89-a58adade1bfb.jpg)
and here when i run find files, you can see the output of the `Telescope find_files` command at the bottom
![mié 07 sep 2022 12:16:01 -05](https://user-images.githubusercontent.com/53343401/188940240-f81235d2-e3fc-4e98-9b92-0ec56f50a64d.jpg)

**Comment by goolord** [comment, marked as answer]
replacing the button with  `button("SPC f f", "  Find file", "<CMD>silent Telescope find_files<CR>")` should work for now, but i'll work on a real fix for this
